crates/sui-graphql-rpc/src/context_data/db_backend.rs

```diff
@@ -30,12 +34,15 @@ pub(crate) type BalanceQuery<'a, DB> = BoxedSelectStatement<
     objects::dsl::coin_type,
 >;
 
+pub type QueryableEpochInfoType<DB> = SqlTypeOf<AsSelect<QueryableEpochInfo, DB>>;
```

Comment:
You can probably leave this as `pub(crate)` rather than fully exported.

```suggestion
pub(crate) type QueryableEpochInfoType<DB> = SqlTypeOf<AsSelect<QueryableEpochInfo, DB>>;
```